lib/src/migrators/module.dart

```diff
@@ -349,14 +343,6 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
     return forwards.join('');
   }
 
-  /// Immediately throw a [MigrationException] when a missing dependency is
-  /// encountered, as the module migrator needs to traverse all dependencies.
-  @override
-  void handleMissingDependency(Uri dependency, FileSpan context) {
```

Comment:
Since this method only existed because the `_ModuleMigrationVisitor` needed to override it, it probably make sense to eliminate it from `MigrationVisitor`.

Reply:
Done.